### PR TITLE
Radiator Catalog Info

### PR DESCRIPTION
**Pull request by voodooGQ**
This adds a Catalog Info YAML document for the [Radiator](https://github.com/Kajabi/radiator) project. For more details about the project either reach out to @voodooGQ or read the Golden Path documentation [here](https://radiator.staging.kajabi.farm/docs/default/component/radiator-site/#kajabi-radiator).

Below is a checklist of details that need to be answered prior to us merging this in. When time permits please add a comment to this PR with the answers:

- [ ] Do we have a brief description of this service?
- [ ] Are there any relevant "tags" we could give this service?
- [ ] Do we have any Datadog dashboards?
- [ ] Do we have a Honeybadger query for this component?
- [ ] Do we have a Sentry query for this component?
- [ ] Do we have an OpsGenie service for this component?
- [ ] Are there any Snyk projects for this component?
  - [ ] If so could you please provide the Project ID (Found in Settings) 
- [ ] Are there any Confluence documents that reference this?
- [ ] Is there a LucidChart?
- [ ] Do we have a list of Feature Flags or Env vars?
  - [ ] If so this information is needed: https://radiator.staging.kajabi.farm/docs/default/component/radiator-site/software-catalog/well-known-metadata/#environment-variables
- [ ] Does it depend on any internal services?
- [ ] Does it depend on any external services?
- [ ] Does it depend on any resources (postgres database, redis, ect.)

### CODE REVIEW

**Review by sdhull — COMMENTED** [review]
## Description
Cornerstone Product is a product theme. It's offered as boilerplate starting point for 3rd party theme developers [here](https://themes.kajabi.com/liquid/getting-started/theme-types). There is no "release" for the Cornerstone Product Theme, as it isn't installable directly in Kajabi Admin.

## Tags
- theme
- themes
- cornerstone-theme
- theme-product

## Does it depend on any internal services?
The monolith's Liquid Rendering and Theme Editing & Management functionality

## Everything else
N/A

**Comment by voodooGQ** [comment]
@sdhull, ready for review. Thanks!

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  tags:
+    - theme
+    - themes
+    - cornerstone
+    - cornerstone-theme
+    - theme-product
+    - product
+    - node
+    - js
+    - javascript
```
Comment (sdhull) [inline catalog-info.yaml:16]:
```suggestion
    - javascript
    - liquid
```

**Review by sdhull — APPROVED** [review]
LGTM. I realized it makes sense to tag with `liquid` as well

**Pull request merged**